DemoApp/StreamChat/StreamChatWrapper+DemoApp.swift

```diff
@@ -45,12 +46,17 @@ extension StreamChatWrapper {
         Components.default.channelListSearchStrategy = .messages
 
         // Customize UI components
+        Components.default.attachmentViewCatalog = DemoAttachmentViewCatalog.self
```

Comment:
Not in the scope of this PR, but I think the demo app becomes very complex in terms of different configurations. Might be hard for integrators to use it as an example for implementation.

Reply:
Yeah, something we might need to discuss in our tech forum